### ISSUE TITLE
How prevent tag deletion?

### ISSUE THREAD

**Issue by AarneHuttunen** [yabwe/medium-editor#1447]
Basically what I'm looking for is a way to prevent the deletion of an html element. Is there a way to do it? In the example, the button is a single anchor element.

![tag deletion](https://user-images.githubusercontent.com/3826260/40578016-c528cfae-60c1-11e8-9c86-ccb9d737707d.gif)

**Comment by AarneHuttunen** [comment]
In the example, I had the editor initialised on body. I then changed to initialisation on elements with .editable class. This worked better, but the elements could not be edited after they were empty. This was fixed by adding a </br> tag inside the anchor element.

**Comment by marcosignacio** [comment]
Hey, 

You can do it this way:

var active_item;

$(window).on('click', function(event) {
				active_item=undefined;
			
			});
$(".your_element_selector").on('click', 'span', function(e){
				active_item= e.currentTarget;
				event.stopPropagation();
			});
$(".your_element_selector").on('keydown',function(e){
				if (e.which==46){	
					if ($(active_item).html().length<2) {
						e.preventDefault();
						$(active_item).html('');
						return false;
					}
				}
});


Basically the idea is:

1) When you tag get focus/clicked, save a reference in a var.
2) When DEL key is pressed, if that tag is defined and its content length is <2, abort event.

Let me know.

Cheers
